version.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# Package Releases Job
+# Copyright(C) 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This file carries the version of the cve update."""
+
+__version__ = "0.2.0"
```

Comment:
Any reason why this goes outside of `app.py`?

Reply:
No, just separated because in most other repositories `version` was in a separate file, but I will put it back :smile: